src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -61,25 +62,32 @@ export default function PrivateMenu() {
   }
 
   const title = `${session?.user?.name}\n(${session?.user?.email})`
-  const lettersAvatar = `${session.user?.given_name?.charAt(0) || session.user?.email?.charAt(0)}${
-    session.user?.family_name?.charAt(0) || session.user?.email?.charAt(1)
-  }`.toUpperCase()
+
+  const getAvatar = () => {
+    const { data: user } = getCurrentPerson(Boolean(router.query?.register))
+    if (user) return user.user.picture as string
+  }
```

Comment:
You shouldn't be calling getCurrentPerson here. Use the hook we have `useCurrentPerson` that internally calls that function, or is there a reason you need the `getCurrentPerson` call directly?

Reply:
I was not sure about whether i will need to ever pass truthy parameter so i added getCurrentPerson because it seemed 
more configurable and saw we use it on another page as well. Now it seems i can call the hook instead.

Reply:
@MartinHristovv Try to use hooks whenever possible and use direct API calls only within the hooks.